easybuild/easyconfigs/w/WIEN2k/WIEN2k-23.2-intel-2021b.eb

```diff
@@ -0,0 +1,78 @@
+name = 'WIEN2k'
+version = '23.2'
+
+homepage = 'http://www.wien2k.at/'
+description = """The program package WIEN2k allows to perform electronic structure calculations of solids
+using density functional theory (DFT). It is based on the full-potential (linearized) augmented plane-wave
+((L)APW) + local orbitals (lo) method, one among the most accurate schemes for band structure calculations.
+WIEN2k is an all-electron scheme including relativistic effects and has many features."""
+
+toolchain = {'name': 'intel', 'version': '2021b'}
+
+sources = ['%(name)s_%(version)s.tar']
+patches = [
+    '%(name)s-%(version)s_fix_libxc_for_lapw0_mpi.patch',
+    '%(name)s-%(version)s_fix_system_stderr_redirection.patch',
+]
+checksums = [
+    '3a71565a4fb114a2f9af9a1710bd3160781a7566bb2746bef1ba8af372b4d7b0',  # WIEN2k_23.2.tar
+    '1cc480a4824d9185ad5918dfc68c47bcb7826114626c8133d573be901bbdca84',  # WIEN2k-23.2_fix_libxc_for_lapw0_mpi.patch
+    'eb3f987b1f839b9f10c315d3f7a57e181e46bc98bec3a18e5b9942689b75fcc7',
+    # WIEN2k-23.2_fix_system_stderr_redirection.patch
+]
+
+
+download_instructions = """
+WIEN2k can be ordered at http://susi.theochem.tuwien.ac.at/index.html.
+"""
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('Perl', '5.34.0'),
+    ('DFT-D3', '3.2.0'),
+    ('ELPA', '2021.05.001'),
+    ('FFTW', '3.3.10'),
+    ('libxc', '5.1.6'),
+]
+
+osdependencies = [
+    ('glibc-devel', 'libc6-dev'),  # required for libpthread.
+    ('tcsh', '6.24.01')
```

Comment:
This is not how osdeps works, the tuple is supposed to be various names of the package from different OS:es like the one above which is REHL vs DEB names.
So drop the 6.24.01 part.

Reply:
Done.